src/request.rs

```diff
@@ -111,21 +114,12 @@ impl Request {
         self.do_call(Payload::Empty)
     }
 
-    fn do_call(&self, payload: Payload) -> Result<Response> {
+    fn do_call(self, payload: Payload) -> Result<Response> {
```

Comment:
This makes this a breaking change, since we're consuming the Request rather than using a reference to it. Can we make this work with `&self`?

Reply:
It's actually not. This is not `pub`, but used from `call`, `send`, `send_form` etc, and all those are consuming. This change is simply avoiding a clone that was here earlier, but doesn't change the public API.